### PR TITLE
Update mel_ops.py

### PR DESCRIPTION
**Pull request by keunwoochoi**
This update explains the mel scale formula that is used in Tensorflow implementation. This can resolve the reproducability/compatibility issues signal processing people would face when they are working with other signal processing library such as librosa.

### CODE REVIEW

**Comment by keunwoochoi** [comment]
(cc' @rryan :)

File: tensorflow/python/ops/signal/mel_ops.py [hunk 1]
```diff
@@ -102,6 +102,9 @@ def linear_to_mel_weight_matrix(num_mel_bins=20,
   `num_spectrogram_bins` linearly sampled frequency information from
   `[0, sample_rate / 2]` into `num_mel_bins` frequency information from
   `[lower_edge_hertz, upper_edge_hertz]` on the [mel scale][mel].
+  
+  The implementation follows a mel scale formula, 2595 * log10(1 + f/700), where
```
Comment (rryan) [inline tensorflow/python/ops/signal/mel_ops.py]: how about using TeX to typeset the formula? 
I'd also suggest leading with the HTK bit? Something like:

```
This function follows the [Hidden Markov Model Toolkit (HTK)](http://htk.eng.cam.ac.uk/) convention, defining the mel scale in terms of a frequency in hertz according to the following formula:

$$\textrm{mel}(f) = 2595 * \textrm{log}_{10}(1 + \frac{f}{700})$$
```

**Review by rryan — CHANGES_REQUESTED** [review]
thanks!

**Comment by mihaimaruseac** [comment]
Hi. Please make the PR against master as the release branches are only updated during releases.

File: tensorflow/python/ops/signal/mel_ops.py (hunk 1 above)
Comment (keunwoochoi) [reply tensorflow/python/ops/signal/mel_ops.py]: Will do something like that in another PR. Thanks :)

**Pull request closed without merging**